app/src/main/java/bean/pwr/imskamieskiego/MapActivity.java

```diff
@@ -355,5 +355,42 @@ public boolean onMenuItemClick(MenuItem item) {
         });
     }
 
+    @Override
+    public void setChangeFloorButtonCoords(int barHeight, int resumeCounter) {
+        float margin = 20;
```

Comment:
A co, jeżeli w layoucie zmienię odstęp? Tutaj też będę musiał go zmieniać osobno. To jest złe rozwiązanie.